### PR TITLE
[FEAT] 목표별 히스토리 리스트 조회, 히스토리별 성공률 조회

### PR DESCRIPTION
**Pull request by sueelly**
## #️⃣ Related Issue

Linked Feature backlog: 

## 📝 Purpose of PR

Please briefly explain your work in this PR.

## Contents

List of work items performed.

## Checklist

-   [ ] Does commit messages follow the convention?
-   [ ] Are variable names brief but descriptive?
-   [ ] Is the code clean and easy to understand?
-   [ ] If a new feature has been added, or a bug fixed, has a test been added to confirm good behavior?
-   [ ] Does the test successfully test edge and corner cases?

## 💬 Review Requirements (Optional)

Is there anything you can praise about this PR? Start with praise.
Make a review, leaving kind comments and suggesting changes where needed (to resolve the above).

### CODE REVIEW

File: src/main/java/ac/dnd/dodal/application/plan_history/repository/PlanHistoryRepository.java [hunk 1]
```diff
@@ -1,11 +1,38 @@
...
 import org.springframework.stereotype.Repository;
 
 import ac.dnd.dodal.domain.plan_history.model.PlanHistory;
+import ac.dnd.dodal.ui.plan_history.response.HistoryResponse;
 
 @Repository
 public interface PlanHistoryRepository extends JpaRepository<PlanHistory, Long> {
-    
+
+    @Query("SELECT new ac.dnd.dodal.ui.plan_history.response.HistoryResponse("
```
Comment (5hseok) [inline src/main/java/ac/dnd/dodal/application/plan_history/repository/PlanHistoryRepository.java:15]: 오 이렇게 JPQL을 사용할 수도 있군요!
하나 배워갑니다~

File: src/main/resources/db/migration/V15__change_statistics.sql
Comment (5hseok) [inline src/main/resources/db/migration/V15__change_statistics.sql:1]: 그 전 버전에서 14를 안 쓰게 되어서 V14로 변경해주시면 감사하겠습니다!
V14가 빈 내용이어서 삭제되었어요

**Review by 5hseok — CHANGES_REQUESTED** [review]
